go.mod

```diff
@@ -6,13 +6,13 @@ require (
 	github.com/containerd/containerd v1.7.2
 	github.com/containernetworking/cni v1.1.2
 	github.com/containernetworking/plugins v1.3.0
-	github.com/containers/common v0.55.1
+	github.com/containers/common v0.55.1-0.20230707133902-166256636e4c
 	github.com/containers/image/v5 v5.26.1
 	github.com/containers/ocicrypt v1.1.7
-	github.com/containers/storage v1.48.0
+	github.com/containers/storage v1.48.1-0.20230707125135-6dc2de36ca86
```

Comment:
This common and storage version should only be going into Podman v4.6 at this point.  We need to pull the upstream for main, 0.56.0-dev for common and 1.49.0-dev for storage

Reply:
and Buildah 1.31 too, not upstream.